lookout/style/typos_checker/README.md

```diff
@@ -1,53 +1,86 @@
 # Typos checker
 
-Steps to reproduce the demo as it was seen during the project presentation.
+**Typos checker** is a module for **Style Analyzer** for **[lookout](https://github.com/src-d/lookout)**, which points out typos in code comments when they're introduced or modified by a Pull Request.
 
-PR and issues are welcomed ;)
+# How to run it
 
-## how to reproduce
+You can follow these steps to reproduce the demo, as it was seen during the presentation of the hackathon project.
+
+PR and issues are very welcomed ;)
+
+## Run lookout
+
+You will need to have [Go](https://golang.org/doc/install#install) and [docker-compose](https://docs.docker.com/compose/install/)
+
+### 1. Install [lookout](https://github.com/src-d/lookout)
 
-### 1. Install [lookout](https://github.com/src-d/lookout) locally
 ```shell
-$ go get github.com/src-d/lookout
+$ go get github.com/src-d/lookout/...
 ```
 
-### 2. From lookout directory, let's run the app dependencies using `docker compose`
+### 2. Run lookout dependencies using `docker compose`
+
+**lookout** requires a running [bblfsh](https://doc.bblf.sh) server and a [Postgresql](https://www.postgresql.org/) database; to get them you can run from **lookout** directory:
+
 ```shell
 $ docker-compose up bblfsh postgres
 ```
-and initialize lookout database
+
+and then initialize **lookout** database
+
 ```shell
 $ lookoutd migrate
 ```
 
 ### 3. Run lookout server
+
 ```shell
 $ lookoutd serve --github-token <gh-token> --github-user <user> <repository>
 ```
+
 where:
 - `user` is the github user handler that will post the comment,
 - `gh-token` is [the token of the user](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line) that will post the comment,
 - `repository` is the repository url that will be watched for new PRs (it MUST be public)
 
-### 4. Change analyzer `config.yml`; exposed port should be `10302`
+## Run Style Analyzer
+
+You will need to have [Python](https://www.python.org/downloads/) (between `3.5` and `3.7`)
```

Comment:
suggestion: If I'm not wrong 3.7 is the latest version so something like higher than 3.5 should be fine as well

Reply:
I just followed @vmarkovtsev recommendations at [style-analyzer/getting_started](https://github.com/src-d/style-analyzer/blame/master/lookout/core/doc/getting_started.md#L8) guide.
He might be not confident about back compatibility on new Python versions. Would you agree on that?

Reply:
Before I add the comment, I checked that the latest version of Python is 3.7.0, so I thought it's not necessary. However, I don't know how often Python releases a new version...